linera-core/src/client.rs

```diff
@@ -765,6 +814,7 @@ where
         Ok(certificate)
     }
 
+    #[tracing::instrument(level = "trace", skip(committee, proposal, value))]
```

Comment:
Optional: the proposal and the value hash might be useful here.

```suggestion
    #[tracing::instrument(
        level = "trace",
        skip(committee, value),
        fields(certificate_hash = value.hash())
    )]
```